dali/base/dadfs.cpp

```diff
@@ -2738,6 +2738,11 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
         return *t;
     }
 
+    IPropertyTree &queryHistory() const
+    {
+        return *root->queryPropTree("History");
```

Comment:
Previously discussed (https://github.com/hpcc-systems/HPCC-Platform/pull/9197#discussion_r81319781)
If you're forcing History node to exist and returning a reference.
You need to check here too and create if missing (will only apply to legacy) - and comment as to why.